### ISSUE TITLE
0% Accuracy for 0.02 vx and 5 cm 5 degree metrics

### ISSUE THREAD

**Issue by Pai-Sho** [Microsoft/singleshotpose#49]
I have created my own training dataset and set it up in the exact same way as the included LINEMOD objects. I am able to successfully train to a reasonably high degree of accuracy (82%) but only for the 5 pixel 2D projection metric. The other metrics are always 0%

```
2018-10-10 16:55:40    Mean corner error is 5.215204
2018-10-10 16:55:40    Acc using 5 px 2D Projection = 81.28%
2018-10-10 16:55:40    Acc using 0.02 vx 3D Transformation = 0.00%
2018-10-10 16:55:40    Acc using 5 cm 5 degree metric = 0.00%
2018-10-10 16:55:40    Translation error: 0.813218, angle error: 2.663330
```

However, if I change my cfg file to point to any one of the LINEMOD object's .ply instead of the .ply for the actual object I'm training with, all 3 metrics shoot up to the 95% range. How is this possible? Is there a problem with my .ply file? When I run the validation script, the bounding box predictions for my object appear to be extremely accurate so I don't understand how the accuracy is so low for the 3D metrics

**Comment by Pai-Sho** [comment]
After looking further it looks like there are some significant differences between the format of my .ply files and yours. Are there some specific settings we need to generate the ply files with? How were your files generated?

**Comment by SonOfCoding** [comment]
I have the same problem, how do you fix this problem?

**Comment by Pai-Sho** [comment]
For me it was a scaling issue, my .ply files used millimeters as units but the code expects units of meters. Check the other ply files included with the implementation against yours to make sure it is scaled correctly

**Comment by xkevincode** [comment]
I also train with my own ply file and get the same 0 accuracy for 3D. But the bounding box is also wrong after 700 epochs training. my training image is 160*120, much smaller than the image in the original file. The yellow cup is my object and the result is like following:
![figure_1](https://user-images.githubusercontent.com/22719841/49422352-83b80900-f7ce-11e8-866f-483c3c765dbf.png)
